amplify/backend/function/teamRouter/src/index.py

```diff
@@ -79,24 +78,26 @@ def getEntitlements(userId, groupIds):
     for id in [userId] + groupIds:
         if not id:
             continue
-        entitlement = get_entitlements(id)
-        if "Item" not in entitlement.keys():
+        entitlements = get_entitlements(id)
+        if "Items" not in entitlements.keys():
             continue
-        duration = entitlement['Item']['duration']
-        if int(duration) > maxDuration:
-            maxDuration = int(duration)
-        policy = {}
-        policy['accounts'] = entitlement['Item']['accounts']
-        
-        for ou in entitlement["Item"]["ous"]:
-            data = list_account_for_ou(ou["id"])
-            policy['accounts'].extend(data)
+
+        for entitlement in entitlements["Items"]:    
+            duration = entitlement['duration']
+            if int(duration) > maxDuration:
```

Comment:
I belive this is an error if there happens to be an entitlement which covers multiple groups or users e.g. the query has more than one result for the entity_id. If so, the maxDuration will always be the highest found in the result.

Reply:
We have a bug report internally. As soon as we have a working solution for this, I will share this.

Reply:
Bug was fixed and the fix committed to this pull request. Thanks for your help!